recipe/meta.yaml

```
@@ -1,5 +1,5 @@
 {% set version = "v0.5" %}
-{% set build = 0 %}
+{% set build = 3 %}
```

build should be 1, it should only be incremented for each version on the main feedstock that builds
```suggestion
{% set build = 1 %}
```

---

implemented as suggested